`sfons`/`fontstash` import doesn't bring in C code

**V version:** 
OS: linux, Ubuntu 18.04.6 LTS
Processor: 4 cpus, 64bit, little endian, Intel(R) Core(TM) i7-5600U CPU @ 2.60GHz
CC version: cc (Ubuntu 7.5.0-3ubuntu1~18.04) 7.5.0

getwd: ~/Projects/v
vmodules: ~/.vmodules
vroot: ~/Projects/v
vexe: ~/Projects/v/v
vexe mtime: 2022-01-06 12:46:28
is vroot writable: true
is vmodules writable: true
V full version: V 0.2.4 fb66ec7

Git version: git version 2.17.1
Git vroot status: weekly.2021.52.1-67-gfb66ec7c (2 commit(s) behind V master)
.git/config present: true
thirdparty/tcc status: thirdparty-linux-amd64 3654d6a8

**What did you do?**
Compile the following with `v run .`:
```v
module main

import fontstash
import sokol.sfons

const default_font = font_path()

struct FontCache {
mut:
    context &fontstash.Context
}

pub fn (mut fc FontCache) free() {
    unsafe { sfons.destroy(fc.context) }
}

fn font_path() string {
    return '/tmp'
}

fn main() {
    println('Great success')
}
```

**What did you expect to see?**
`Great success`
 
**What did you see instead?**
```
==================
/dev/shm/v_1000/sfons_bug.6727889927934991907.tmp.c:11615: warning: implicit declaration of function 'sfons_flush'
tcc: error: undefined symbol 'sfons_create'
tcc: error: undefined symbol 'sfons_destroy'
tcc: error: undefined symbol 'sfons_rgba'
tcc: error: undefined symbol 'sfons_flush'
...
==================
(Use `v -cg` to print the entire error message)

builder error: 
==================
C error. This should never happen.

This is a compiler bug, please report it using `v bug file.v`.
```

--- a/vlib/sokol/sfons/sfons.c.v
+++ b/vlib/sokol/sfons/sfons.c.v
@@ -1,9 +1,10 @@
 module sfons
 
 import fontstash
+import sokol.f
 
 // keep v from warning about unused imports
-const used_import = fontstash.used_import + 1
+const used_import = f.used_import + fontstash.used_import + 1
 
 [inline]
 pub fn create(width int, height int, flags int) &fontstash.Context {

